src/OpenTelemetry.Exporter.Prometheus/PrometheusExporterOptions.cs

```diff
@@ -46,9 +47,29 @@ public class PrometheusExporterOptions
 
         /// <summary>
         /// Gets or sets the prefixes to use for the http listener. Default
-        /// value: http://*:80/.
+        /// value: http://localhost:9184/.
         /// </summary>
-        public IReadOnlyCollection<string> HttpListenerPrefixes { get; set; } = new string[] { "http://*:80/" };
+        public IReadOnlyCollection<string> HttpListenerPrefixes
+        {
+            get => this.httpListenerPrefixes;
+            set
+            {
+                _ = value ?? throw new ArgumentNullException(nameof(this.httpListenerPrefixes));
```

Comment:
Could use `Guard.ThrowIfNull` here

Reply:
I think we can do that! I did see your PR on cleaning up sanity checks with the Guard class. I'll update this.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2818